mula/scheduler/rankers/boefje.py

```diff
@@ -31,11 +31,11 @@ def rank(self, obj: Any) -> int:
         grace_period = timedelta(seconds=self.ctx.config.pq_grace_period)
 
         # New tasks that have not yet run before
-        if obj.prior_tasks is None or not obj.prior_tasks:
+        if obj.latest_task is None or not obj.latest_task:
```

Comment:
Just to confirm: can the `latest_task` property ever be a falsy value other than `None`?

Reply:
yes it would be possible to be none if there aren't any prior tasks available